Content for the enhanced line pragma

BillWagner opened the pull request:
Fixes #25674

This PR is also small. The main impact of this feature is that Razor debugging is better.

BillWagner committed:
Content for the enhanced line pragma

Fixes #25674

This PR is also small. The main impact of this feature is that Razor debugging is better.

BillWagner committed:
build warning

gewarren reviewed (commented):
This was a bit hard for me to grasp. I'd suggest not using the number 10 for both the end line and column offset, so it's clear that they're not related.

BillWagner committed:
Apply suggestions from code review

Co-authored-by: Genevieve Warren <24882762+gewarren@users.noreply.github.com>

BillWagner committed:
respond to feedback

Update example to ensure different values.

Now, all columns and lines are different values.

gewarren reviewed (approved):
I'm still not sure how line 5 column 60 relates to the single line of code in the example, but perhaps it will be clear to readers more familiar with this.

BillWagner committed:
add links to speclet.

BillWagner commented:
Thanks @gewarren 

I've made the updates. I agree this one is a bit hard to grasp. The end of the span gets complicated very quickly. I also added links to the feature spec, which provides further examples.

BillWagner committed:
typo.
